pkg/controllers/edgeconnect/controller.go

```diff
@@ -447,12 +447,12 @@ func (controller *Controller) reconcileEdgeConnectProvisioner(ctx context.Contex
 		}
 	}
 
-	k8sHostname := controller.k8sAutomationHostPattern(edgeConnect.Name, edgeConnect.Namespace, edgeConnect.Status.KubeSystemUID)
+	k8sAutomationHostPattern := controller.k8sAutomationHostPattern(edgeConnect.Name, edgeConnect.Namespace, edgeConnect.Status.KubeSystemUID)
```

Comment:
(I know this is more of a comment for the previous change 😬 )

Why isn't `k8sAutomationHostPattern` a method on the `EdgeConnect` type ? 

(you are only using parts of `EdgeConnect` so it would be a nice abstraction)
(also, having it as a method of the `controller` makes little sense, when it doesn't use anything from the `controller`)